src/domain/notifications/v2/entities/device-type.entity.ts

```diff
@@ -1,5 +1,5 @@
 export enum DeviceType {
-  ANDROID = 'ANDROID',
-  IOS = 'IOS',
-  WEB = 'WEB',
+  Android = 'ANDROID',
+  Ios = 'IOS',
+  Web = 'WEB',
```

Comment:
Because in `src/routes/notifications/v1/entities/register-device.dto.entity.ts` we are reading DeviceType from `src/domain/notifications/v1/entities/device.entity.ts`.
1. It's better to read from one place.
2. For now, I’ve updated the keys to align with those in the other enum.